draft-ietf-scitt-architecture.md

```diff
@@ -624,12 +623,12 @@ Append-Only:
 
 Non-equivocation:
 
-: there is no fork in the Append-only Log.
+: there is no fork in the Verifiable Data Structure.
 Everyone with access to its content sees the same ordered collection of Signed Statements and can check that it is consistent with any Receipts they have verified.
 
 Replayability:
 
-: the Append-only Log includes sufficient information to enable authorized actors with access to its content to check that each included Signed Statement has been correctly registered.
+: the Verifiable Data Structure includes sufficient information to enable authorized actors with access to its content to check that each data structure representing each Signed Statement has been correctly registered.
```

Comment:
```suggestion
: the Append-only Log includes sufficient information to enable authorized actors with access to its content to check that each data structure representing each Signed Statement has been correctly registered.
```

The VDS only has digests, it does not contain enough information for an auditor to get roots of trust, registration policies etc. It's only enough to check consistency, not registration.

Reply:
Is this question going beyond the change from `Append-only Log` to `Verifiable Data Structure`?

Reply:
Calling @robinbryce for ⬆️

Reply:
The MMR does not contain that information. My objection is to the implication that the TS must be responsible for making the information available to auditors. This is inherently centralizing and makes verifiabilty depend on the continued good operations of the TS.

A requirement that the TS must *define* how the information is available to auditors is fine by me. Its a one line change to make that work I think.

The move to replace "append only log" as a term with a "Registered Sequence of Signed Statements" helps a lot here I think - see https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/351/files#r1975294713

Reply:
I'm ok with ""Registered Sequence of Signed Statements""

This does not dictate implementation expectations the way that logs do.

Reply:
+1

Reply:
Deferring to #348